src/controller/AddNoteForm.tsx

```diff
@@ -69,11 +69,11 @@ const AddNoteForm = (props: Props) => {
         </form>
       </ActionModal>
       {rowData.snoozeInformation ? (
-        <UsaButton onClick={openModal} buttonStyle="outline">
-          Add Note
+        <UsaButton onClick={openModal} buttonStyle="unstyled">
+          Add A Note
         </UsaButton>
       ) : null}
-      {/*todo remove ternary statement*/}
+      {/*TODO remove ternary statement*/}
```

Comment:
😆 

Does this trigger a warning of some kind, or is it just for convenience? (In Java-land, it's a fatal style error, but that's a convention, obviously.)

Reply:
When we add linting to our CI we could fail on this with https://eslint.org/docs/rules/no-warning-comments